device-types/QNAP/TS-832PXU-RP.yaml

```diff
@@ -0,0 +1,46 @@
+---
+manufacturer: QNAP
+model: TS-832PXU-RP
+slug: qnap-ts-832pxu-rp
+part_number: TS-832PXU-RP
+u_height: 2
+airflow: front-to-rear
```

Comment:
To have the most complete data possible could you add the `weight` and `weight-unit` values

Reply:
Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/2e7c8da0c02c2b630fe4583e36af1b60ee9702b3